django/src/pages/views.py

```diff
@@ -114,3 +114,10 @@ def profiles(request: HttpRequest, username: str):
 		if (target.id is game.user1.id and game.score1 > game.score2) or (target.id is game.user2.id and game.score2 > game.score1):
 			win += 1
 	return create_response(request, 'profiles.html', {'target': target, 'games': games, 'wins': win}, title=f"{target.username} Profile")
+
+@require_GET
+def settings(request: HttpRequest):
+	if not request.user.is_authenticated:
```

Comment:
already done in the `create_response` with the `need_authentication` to True

Reply:
adressed in : [d42776c](https://github.com/TheSecretOrganization/ft_transcendence/pull/247/commits/d42776cdca442763caa7de3c21eacf49a904bcf6)